packages/x-scheduler/src/primitives/day-grid/cell/DayGridCell.test.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import { DayGrid } from '@mui/x-scheduler/primitives/day-grid';
+import { createSchedulerRenderer, describeConformance } from 'test/utils/scheduler';
+
+describe('<DayGrid.Cell />', () => {
+  const { render } = createSchedulerRenderer();
+
+  describeConformance(<DayGrid.Cell />, () => ({
+    refInstanceof: window.HTMLDivElement,
+    render(node) {
+      return render(
+        <DayGrid.Root>
+          <DayGrid.Row key="week-1">
+            <DayGrid.Cell key="day-1">{node}</DayGrid.Cell>
```

Comment:
You should not have the `<DayGrid.Cell>` here, only the wrapper. The component you are testing is already inside the `node`

Reply:
Whoops, my bad. 🙈